tests/e2e/performance/io_workload/test_fio_benchmark.py

```diff
@@ -199,7 +199,6 @@ def setting_io_pattern(self, io_pattern):
         """
         if io_pattern == "sequential":
             self.crd_data["spec"]["workload"]["args"]["jobs"] = ["write", "read"]
-            self.crd_data["spec"]["workload"]["args"]["iodepth"] = 1
```

Comment:
why this line removed ?

Reply:
sequential fio runs gave considerable degradaton upon running with iodepth=1  and pod count of 9 whereas with pod count 9 and iodepth =16 higher iops were achieved. Hence for both sequential and random workload pod count of 9 and iodepth of 16 is used. (https://docs.google.com/document/d/1vTnBv0j5-Tnle3KdG3PU36G_N1SbjGJRiwmBb07ZYlg/edit#)

Reply:
@pintojoy so are we removing this line? or we would like to increase the value from 1 to 16, iodepth default value is 1 in fio.

Reply:
@ramkiperiy Previously we were running with following configuration:
 server pods =51 iodepth =16 for random workload and server pods =51 iodepth =1 for sequential workload.

But with server pods=51 we were over-stressing the system, Hence pod count 9 was chosen and with pod count =9 and iodepth =16 random workload gave improved iops value whereas with sequential(pod count =9 and iodepth=1) degradation was seen, which again goes away when iodepth is increased to 16. Hence to maintain uniformity, pod count =9 and iodepth =16 is chosen for both sequential and random workloads.

Reply:
ok, My question was where are we setting it ? noticed it's in benchmark_fio.yaml param list, so default it would be 16